docs/source/userguide/plugin_development/algorithm.rst

```diff
@@ -74,7 +74,8 @@ Copy and paste this code into your algorithm file (feel free to remove the comme
 
     def call(
         xobj, # Xarray dataset holding xarrays
-        variables, # list of required input variables for algorithm. Note: Python lists are ordered, so you can count on your list of variables being in the order in which you define them in your product plugin variables
```

Comment:
This needs manual reformatting

Reply:
done

Reply:
Did this require manual reformatting because comments don't get wrapped? Which package is handling the wrapping in embedded code blocks?

Reply:
This needed manual reformatting because I wanted the comment on a new line instead of simply wrapped. Right now `pinkrst` just wraps lines that are too long. 99% of the time this isn't a problem, but if it's in a code block that's an issue. 

Our line length maximum (120) leaves enough room for a few indents AND our maximum code length (88) so any code that would be auto-wrapped would have been written for the docs explicitly soooo I'm of the opinion we don't have to worry about it too much.

Reply:
Sounds good to me!